modules/certificate/certificate.service.js

```diff
@@ -235,6 +239,31 @@ class CertificatesService extends FilterHelper {
     ).exec();
   }
 
+  async gitftCertificateToEmail(id, email, oldEmail, language) {
+    const certificate = await this.getCertificateById(id);
+    certificate.email = email;
+    certificate.save();
+
+    const dateEnd = certificate.dateEnd.toLocaleDateString();
+    const dateStart = certificate.dateStart.toLocaleDateString();
+
+    await sendEmail(email, RESIVE_GIFT_SERTIFICATE, {
+      certificateName: certificate.name,
+      certificateValue: certificate.value,
+      dateEnd,
+      language,
+    });
+    await sendEmail(oldEmail, SEND_GIFT_CERTIFICATE, {
+      certificateName: certificate.name,
+      certificateValue: certificate.value,
+      dateStart,
+      dateEnd,
+      language,
+    });
+
+    return true;
```

Comment:
The graphql declaration says this function is supposed to return a `CertificateResult`, which is not just a boolean value. Why did you return this?

Reply:
fixed